app/elements/qea-property-analyzer/qea-property-analyzer.html

```diff
@@ -0,0 +1,138 @@
+<link rel="import" href="../../bower_components/polymer/polymer.html">
+<link rel="import" href="../../bower_components/hydrolysis/hydrolysis-analyzer.html">
+<link rel="import" href="../../behaviors/qea-inspect-behavior.html">
+
+<!-- use this element as a library to analize a polymer elements -->
+<dom-module id="qea-property-analyzer">
+  <template>
+    <style>
+      :host {
+        display: block;
+      }
+    </style>
+
+    <hydrolysis-analyzer id='hydro'
+         src=[[src]]
+         analyzer={{_analyzer}}
+         transitive=[[recursively]]
+         loading={{loading}}
+         clean>
+    </hydrolysis-analyzer>
+
+  </template>
+  <script>
+  (function () {
+    'use strict'
+
+    let MyBehavior = {};
+    class qeaPropertyAnalyzer {
+
+      get behaviors() {
+        return [qeaBehaviors.inspectBehavior];
+      }
+
+      beforeRegister(){
+        this.is = 'qea-property-analyzer'
+        this.properties = {
+          // path for the element to be analyzed
+          src: {
+            type: String
+          },
+          // result of the analisis
+          _analyzer: {
+            type: Object,
+            notify: true
+          },
+
+          recursively: {
+            type: Boolean,
+            value: false
+          },
+          loading: {
+            type: Boolean,
+            notify: true
+          },
+          props: {
+            type: Array,
+            computed: '_computeProps(_analyzer)',
+            value: ()=>[],
+            notify: true
+          }
+
+        }
+      }
+
+      _computeProps(analyzer){
+        if (!analyzer || Object.keys(analyzer).length === 0
+            || !analyzer.elements || analyzer.elements.length == 0) {
+          return []
+        }
+        return this.getProperties(analyzer);
+      }
+
+      // analyze element passed as parameter using hydrolysis and return
+      // the result as Promise
+      scanElement(tagName){
+        return new Promise(
+          (resolve, reject)=>  {
+            this.src = this.resolvePath(tagName)
+            this.$.hydro.analyze()
+            this.addEventListener('_analyzer-changed', ()=> {
+              // console.log('analy',  this._analyzer);
+              if (this._analyzer == null) {
+                reject(new Error('impossible to analyze element '+ this.src))
+              }
+              resolve(this._analyzer)
+            })
+          }
+        )
+      }
+
+      // return the list of behaviors of the element analyzed
+      getBehaviors(analyzer){
```

Comment:
You have a getter for behaviours and then you have the `getBehaviors` function. What's their difference? As an interface, isn't this confusing?